environments/development/inventory.ini

```diff
@@ -98,3 +98,5 @@ swap_size=100M
 
 [control:vars]
 hostname='control'
+
+[barman]
```

Comment:
What do you think about calling this `pg_barman`, to show someone new to it that it's pg-related?